packages/evm/src/evmJournal.ts

```diff
@@ -0,0 +1,283 @@
+import { Hardfork } from '@ethereumjs/common'
+import {
+  Address,
+  RIPEMD160_ADDRESS_STRING,
+  bytesToHex,
+  stripHexPrefix,
+  toBytes,
+} from '@ethereumjs/util'
+import { debug as createDebugLogger } from 'debug'
+
+import type { Common, EVMStateManagerInterface } from '@ethereumjs/common'
+import type { Account } from '@ethereumjs/util'
+import type { Debugger } from 'debug'
+
+type WarmSlots = Set<string>
+type AddressString = string
+type SlotString = string
+
+type Journal = Map<AddressString, WarmSlots>
+
+/**
+ * Journal Diff Item:
+ * Index 0: remove warm address
+ * Index 1: remove warm slots for this warm address
+ * Index 2: remove touched
+ */
+
+type JournalDiffItem = [Set<AddressString>, Map<AddressString, Set<SlotString>>, Set<AddressString>]
+//AddressString | Map<AddressString, SlotString>
+type JournalHeight = number
+
+export class EvmJournal {
```

Comment:
Here as well, I would suggest to just name `Journal` and also simplify the file name `evmJournal.ts` -> `journal.ts`.

Reply:
Should I rename `evm.evmJournal` to `evm.journal` as well?

Reply:
I would vote yes! 🙂